includes/common/common-functions.php

```diff
@@ -67,365 +74,9 @@ function getShipmentItems($orderId, $currency, $originCountryCode)
 }
 
 /**
- * @param int  $postId
- * @param null $shipKey
- */
-function updateShipmentKey($postId, $shipKey = null)
-{
-    if (!empty($shipKey)) {
-        update_post_meta($postId, GET_META_MYPARCEL_SHIPMENT_KEY, $shipKey); //Update the shipment key on database
-    } else {
-        add_post_meta($postId, GET_META_MYPARCEL_SHIPMENT_KEY, uniqid()); //Update the shipment key on database
-    }
-}
-
-/**
- * @return array
- */
-function getMyParcelShopList()
-{
-    $shops = [];
-    if (get_option('client_key') && get_option('client_secret_key')) {
-        $getAuth = new MyParcelApi();
-        $api = $getAuth->apiAuthentication();
-        if ($api) {
-            $shops = $api->getShops()->limit(100)->get();
-            usort($shops, function ($a, $b) {
-                return strcmp(strtolower($a->getName()), strtolower($b->getName()));
-            });
-        }
-    }
-
-    return $shops;
-}
-
-/**
- * @desc setting page html
- */
-function prepareHtmlForSettingPage()
```

Comment:
FYI - functions like this have been moved to the PHP file responsible for the settings page